File: sea-shell-service-spring-web-flux/sea-shell-rest-service/src/main/java/org/jesperancinha/shell/webflux/config/SeaShellServiceConfiguration.java
package org.jesperancinha.shell.webflux.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.reactive.config.EnableWebFlux;

/**
 * Created by jofisaes on 03/08/2021
 */
@Configuration
@EnableWebFlux
@ComponentScan("org.jesperancinha.shell.client")
public class SeaShellServiceConfiguration {
}

File: sea-shell-service-immutable/src/main/kotlin/org/jesperancinha/shell/webflux/immutable/config/SeaShellServiceConfiguration.kt
package org.jesperancinha.shell.webflux.immutable.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.web.reactive.config.EnableWebFlux

@Configuration
@EnableWebFlux
@ComponentScan("org.jesperancinha.shell.client")
class SeaShellServiceConfiguration 